### PR TITLE
feat: add Abbreve project link to open_source/projects.json

### LINKED ISSUE #2502: [Added] : Abbreve  project link to Open Source/Projects
### Site URL

https://github.com/Njong392/Abbreve

### Description of the above provided site

Abbreve is an open-source dictionary for slang. With Abbreve, you'll never have to feel left out of a conversation or unsure of the meaning of an abbreviation again.

### Category

Open Source

### Sub Category

Projects

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by sahajj11**
## Fixes Issue
Closes #2502 

## Changes proposed
-Added  Abbreve open source project link to open_source/projects.json.

-Abbreve is an open-source dictionary for slang. With Abbreve, you'll never have to feel left out of a conversation or unsure of the meaning of an abbreviation again.

## Screenshots
linkhub

## Note to reviewers
- No major changes to functionality, only data addition.

### CODE REVIEW

File: package.json
Comment (Jay-Karia) [inline package.json:1]: No need to change the versions of deps

Comment (sahajj11) [reply package.json:1]: Hi, it looks like package.json was flagged by the automated compliance checks, but I can confirm that no changes were made to this file in this PR.

Comment (Jay-Karia) [inline package.json:1]: It's the same issue here.

File: pnpm-lock.yaml
Comment (Jay-Karia) [inline pnpm-lock.yaml:1]: And here

**Review by Jay-Karia — CHANGES_REQUESTED** [review]

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**